javascript-timers/interval.js

```diff
@@ -0,0 +1,13 @@
+const countDown = document.querySelector('.countdown-display');
+let startingNum = 3;
+
+setInterval(() => {
+  if (startingNum === 0) {
+    countDown.textContent = '~earth beeloow us~';
+  } else if (startingNum < 0) {
+    clearInterval();
```

Comment:
FYI💡: You can combine this into one `if statement`! 

Here's an example (with a named function not an arrow function, but both syntaxes work!):
```
function countDown() {
  count--;
  if (count > 0) {
    $countdownDisplay.textContent = count;
  } else {
    $countdownDisplay.textContent = '~Earth Beeeelooowww Us~';
    clearInterval(countdownId);
  }
}
```